### ISSUE TITLE
Use in a server/modpack

### ISSUE THREAD

**Issue by EliteTechnologies5542** [EliteAspect21/morefoodz#3]
I was wondering if I can use this on my server or in a modpack. If so, can I sell it?

**Comment by EliteAspect21** [comment]
Yes, you can use it on your server and in a modpack granted you don't claim it as your own. And if you want to sell a modpack with the mod in it, please contact us at Business@elitetech-usa.com. Our representative will decide and set up terms. At least 5% of revenue must be payed to the company.
